tests/plugins/org.polarsys.capella.test.diagram.tools.ju/src/org/polarsys/capella/test/diagram/tools/ju/testsuites/partial/TitleBlocksToolsTestSuite.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2020 THALES GLOBAL SERVICES.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *  
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.test.diagram.tools.ju.testsuites.partial;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.polarsys.capella.test.diagram.tools.ju.tb.SA_CDBTitleBlockTestCase;
+import org.polarsys.capella.test.framework.api.BasicTestArtefact;
+import org.polarsys.capella.test.framework.api.BasicTestSuite;
+
+import junit.framework.Test;
+
+public class TitleBlocksToolsTestSuite extends BasicTestSuite {
+
+  /**
+   * Returns the suite. This is required to unary launch this test.
+   */
+  public static Test suite() {
+    return new TitleBlocksToolsTestSuite();
+  }
+
+  @Override
+  public List<String> getRequiredTestModels() {
+    return Arrays.asList("TitleBlocksModel");
+  }
+
+  /**
+   * @see org.polarsys.capella.test.framework.api.BasicTestSuite#getTests()
+   */
+  @Override
+  protected List<BasicTestArtefact> getTests() {
+    List<BasicTestArtefact> tests = new ArrayList<BasicTestArtefact>();
+    tests.add(new SA_CDBTitleBlockTestCase());
+    
```

Comment:
add also the test that check preferences (the test TitleBlockPreferencePageTest )